src/PodActions.jsx

```diff
@@ -77,72 +78,98 @@ const PodDeleteModal = ({ con, pod }) => {
 export const PodActions = ({ con, onAddNotification, pod, isPodService }) => {
     const Dialogs = useDialogs();
 
+    const startPod = () => {
+        if (isPodService) {
+            systemctl_spawn(["start", pod.Labels.PODMAN_SYSTEMD_UNIT], con.uid === 0).catch(ex => {
+                const error = cockpit.format(_("Failed to start pod $0"), pod.Name);
+                onAddNotification({ type: 'danger', error, errorDetail: ex.message });
+            });
+        } else {
+            client.postPod(con, "start", pod.Id, {})
+                    .catch(ex => {
+                        const error = cockpit.format(_("Failed to start pod $0"), pod.Name);
+                        onAddNotification({ type: 'danger', error, errorDetail: ex.message });
+                    });
+        }
+    };
+
+    const stopPod = (force) => {
+        if (isPodService) {
+            systemctl_spawn(["stop", pod.Labels.PODMAN_SYSTEMD_UNIT], con.uid === 0).catch(ex => {
+                const error = cockpit.format(_("Failed to stop pod $0"), pod.Name);
+                onAddNotification({ type: 'danger', error, errorDetail: ex.message });
+            });
+        } else {
+            const options = force ? { t: 0 } : {};
+            client.postPod(con, "stop", pod.Id, options)
+                    .catch(ex => {
+                        const error = cockpit.format(_("Failed to stop pod $0"), pod.Name);
+                        onAddNotification({ type: 'danger', error, errorDetail: ex.message });
+                    });
+        }
+    };
+
+    const restartPod = (force) => {
+        if (isPodService) {
+            systemctl_spawn(["restart", pod.Labels.PODMAN_SYSTEMD_UNIT], con.uid === 0)
+                    .catch(ex => {
+                        const error = cockpit.format(_("Failed to restart pod $0"), pod.Name);
+                        onAddNotification({ type: 'danger', error, errorDetail: ex.message });
+                    });
+        } else {
+            const options = force ? { t: 0 } : {};
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2286-f62e03c2-20251017-140225-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)